packages/containers/nri-memory-qos/upstream.yaml

```diff
@@ -0,0 +1,6 @@
+ArtifactHubRepo: nri-plugins
+ArtifactHubPackage: nri-memory-qos
+Vendor: containers
```

Comment:
Is this correct?

Reply:
If you mean the `Vendor:`, then I was not totally sure if I had to write `nri-plugins` (which is the GitHub project name) or the GitHub org (`containers`) that is hosting `nri-plugins` project.

And I didn't find something that would be vendor neutral. nri-plugins isn't a particular vendor owner project but community maintained project. 

Do you think I should replace it by nri-plugins instead?

Reply:
`Vendo` means the company responsible for this software.

Reply:
I assume you are adding this software because you company has a partner relationship with SUSE / Rancher. Right?

Reply:
Actually this project is not tied to a company.